wp-includes/theme-compat/offline.php

```diff
@@ -16,6 +16,7 @@
 <main>
 	<h1><?php esc_html_e( 'Oops! It looks like you&#8217;re offline.', 'pwa' ); ?></h1>
 	<p><?php esc_html_e( 'Please check your internet connection, and try again.', 'pwa' ); ?></p>
+	<p><!-- WP_OFFLINE_COMMENT --></p>
```

Comment:
Instead of `WP_OFFLINE_COMMENT` or `WP_OFFLINE_ERROR_MESSAGE` as I have above, we could have a common string that is used for both offline and 500 error scenarios (per #81). So there could be just `<!--WP_SERVICE_WORKER_ERROR_MESSAGE-->` which is used in all templates regardless.